packages/playground/src/__tests__/testFunction/handler.py

```diff
@@ -0,0 +1,3 @@
+def handle(data):
```

Comment:
can you remove this? I don't think these print statements are required in the sdk

Reply:
I test against the `Hello World!` in the logs test. And, we need something to do in the function anyways.

Reply:
ah, I see it's executed outside the sdk. my mistake